### ISSUE TITLE
Vulnerabilities Tag 1.10.9 

### ISSUE THREAD

**Issue by danielc103** [bitnami/bitnami-docker-airflow#35]
**Description**

Has known critical and high vulnerabilities in latest build.

**Steps to reproduce the issue:**

Run image through scanning utility 

**Describe the results you received:**

```json
{
            "CVE": "NODE-SECURITY-755",
            "Package": "handlebars",
            "Version": "4.0.11",
            "Severity": "critical",
            "Status": "fixed in >=4.0.14 <4.1.0 || >=4.1.2",
            "CVSS": 9
        },
        {
            "CVE": "NODE-SECURITY-1118",
            "Package": "eslint-utils",
            "Version": "1.3.1",
            "Severity": "critical",
            "Status": "fixed in >=1.4.1",
            "CVSS": 9
        },
        {
            "CVE": "NODE-SECURITY-755",
            "Package": "handlebars",
            "Version": "4.0.12",
            "Severity": "critical",
            "Status": "fixed in >=4.0.14 <4.1.0 || >=4.1.2",
            "CVSS": 9
        },
        {
            "CVE": "NODE-SECURITY-1325",
            "Package": "handlebars",
            "Version": "4.0.11",
            "Severity": "high",
            "Status": "fixed in >=3.0.8 <4.0.0 || >=4.5.3",
            "CVSS": 7
        },
        {
            "CVE": "CVE-2019-19919",
            "Package": "handlebars",
            "Version": "4.0.11",
            "Severity": "high",
            "Status": "fixed in >=3.0.8 <4.0.0 || >=4.3.0",
            "CVSS": 9.8
        },
        {
            "CVE": "NODE-SECURITY-813",
            "Package": "js-yaml",
            "Version": "3.12.0",
            "Severity": "high",
            "Status": "fixed in >=3.13.1",
            "CVSS": 7
        },
        {
            "CVE": "CVE-2019-10744",
            "Package": "lodash",
            "Version": "4.17.11",
            "Severity": "high",
            "Status": "fixed in >=4.17.12",
            "CVSS": 9.8
        },
        {
            "CVE": "NODE-SECURITY-1325",
            "Package": "handlebars",
            "Version": "4.0.12",
            "Severity": "high",
            "Status": "fixed in >=3.0.8 <4.0.0 || >=4.5.3",
            "CVSS": 7
        },
        {
            "CVE": "NODE-SECURITY-1324",
            "Package": "handlebars",
            "Version": "4.0.12",
            "Severity": "high",
            "Status": "fixed in >=3.0.8 <4.0.0 || >=4.5.3",
            "CVSS": 7
        },
        {
            "CVE": "NODE-SECURITY-1316",
            "Package": "handlebars",
            "Version": "4.0.12",
            "Severity": "high",
            "Status": "fixed in >=3.0.8 <4.0.0 || >=4.5.2",
            "CVSS": 7
        },
        {
            "CVE": "NODE-SECURITY-1316",
            "Package": "handlebars",
            "Version": "4.0.11",
            "Severity": "high",
            "Status": "fixed in >=3.0.8 <4.0.0 || >=4.5.2",
            "CVSS": 7
        },
        {
            "CVE": "CVE-2019-10744",
            "Package": "lodash",
            "Version": "3.10.1",
            "Severity": "high",
            "Status": "fixed in >=4.17.12",
            "CVSS": 9.8
        },
        {
            "CVE": "CVE-2019-19919",
            "Package": "handlebars",
            "Version": "4.0.12",
            "Severity": "high",
            "Status": "fixed in >=3.0.8 <4.0.0 || >=4.3.0",
            "CVSS": 9.8
        },
        {
            "CVE": "CVE-2018-16487",
            "Package": "lodash",
            "Version": "3.10.1",
            "Severity": "high",
            "Status": "fixed in >=4.17.11",
            "CVSS": 9.8
        },
        {
            "CVE": "CVE-2019-10747",
            "Package": "set-value",
            "Version": "2.0.0",
            "Severity": "high",
            "Status": "fixed in >=2.0.1 <3.0.0 || >=3.0.1",
            "CVSS": 9.8
        },
        {
            "CVE": "CVE-2019-10746",
            "Package": "mixin-deep",
            "Version": "1.3.1",
            "Severity": "high",
            "Status": "fixed in >=1.3.2 <2.0.0 || >=2.0.1",
            "CVSS": 9.8
        },
        {
            "CVE": "NODE-SECURITY-1324",
            "Package": "handlebars",
            "Version": "4.0.11",
            "Severity": "high",
            "Status": "fixed in >=3.0.8 <4.0.0 || >=4.5.3",
            "CVSS": 7
        },
        {
            "CVE": "CVE-2019-10747",
            "Package": "set-value",
            "Version": "0.4.3",
            "Severity": "high",
            "Status": "fixed in >=2.0.1 <3.0.0 || >=3.0.1",
            "CVSS": 9.8
        },
        {
            "CVE": "CVE-2019-16769",
            "Package": "serialize-javascript",
            "Version": "1.5.0",
            "Severity": "moderate",
            "Status": "fixed in >=2.1.1",
            "CVSS": 5.4
        },
        {
            "CVE": "NODE-SECURITY-646",
            "Package": "atob",
            "Version": "2.0.3",
            "Severity": "moderate",
            "Status": "fixed in >=2.1.0",
            "CVSS": 4
        },
        {
            "CVE": "NODE-SECURITY-1300",
            "Package": "handlebars",
            "Version": "4.0.12",
            "Severity": "moderate",
            "Status": "fixed in >=4.4.5",
            "CVSS": 4
        },
        {
            "CVE": "NODE-SECURITY-1300",
            "Package": "handlebars",
            "Version": "4.0.11",
            "Severity": "moderate",
            "Status": "fixed in >=4.4.5",
            "CVSS": 4
        },
        {
            "CVE": "CVE-2019-5428",
            "Package": "jquery",
            "Version": "3.3.1",
            "Severity": "moderate",
            "Status": "fixed in >=3.4.0",
            "CVSS": 4
        },
        {
            "CVE": "NODE-SECURITY-788",
            "Package": "js-yaml",
            "Version": "3.12.0",
            "Severity": "moderate",
            "Status": "fixed in >=3.13.0",
            "CVSS": 4
        },
        {
            "CVE": "NODE-SECURITY-1179",
            "Package": "minimist",
            "Version": "1.2.0",
            "Severity": "low",
            "Status": "fixed in >=0.2.1 <1.0.0 || >=1.2.3",
            "CVSS": 1
        },
        {
            "CVE": "NODE-SECURITY-1179",
            "Package": "minimist",
            "Version": "0.0.8",
            "Severity": "low",
            "Status": "fixed in >=0.2.1 <1.0.0 || >=1.2.3",
            "CVSS": 1
        },
        {
            "CVE": "NODE-SECURITY-1084",
            "Package": "mem",
            "Version": "1.1.0",
            "Severity": "low",
            "Status": "fixed in >=4.0.0",
            "CVSS": 1
        },
        {
            "CVE": "CVE-2018-3721",
            "Package": "lodash",
            "Version": "3.10.1",
            "Severity": "low",
            "Status": "fixed in >=4.17.5",
            "CVSS": 6.5
        }
```

**Describe the results you expected:**

moderate to low vulnerabilities

**Comment by beltran-rubo** [comment]
Hi @danielc103,

Thanks for reporting it. Those vulnerabilities are related to a Node.js-based tool we use to configure the Airflow application https://github.com/bitnami/nami Once the app is configured the tool is NOT running anymore. 

Those CVEs are mainly related for web applications that use those dependencies and this is not the case. I would not consider important in this specific case. In any case, we will work on updating those dependencies.

**Comment by beltran-rubo** [comment]
We updated the Nami tool included in this container (since version 1.10.10-debian-10-r30), you should not get those CVEs in the report.
